package.json

```diff
@@ -34,11 +41,12 @@
   "license": "MIT",
   "scripts": {
     "clean": "rimraf dist",
-    "build": "node build.mjs && npm run build:types",
+    "build": "npm run clean && npx parcel build --no-source-maps && tsc --emitDeclarationOnly",
+    "build:dev": "npm run clean && npx parcel build && tsc --emitDeclarationOnly",
     "build:types": "tsc --declaration --emitDeclarationOnly",
     "postbuild": "npm run check:browser && npm run check:node && npm run test:size",
-    "check:browser": "es-check es2017 --module ./dist/contentful.browser.js",
-    "check:node": "es-check es2017 ./dist/contentful.node.js",
+    "check:browser": "es-check es2017 --module ./dist/contentful.esm_.js",
```

Comment:
where would we generate this output file?

Reply:
Why are we not using   `./dist/contentful.esm.js` here and +1 to Marcos question.

Reply:
@ruderngespra I actually think it's a typo - we don't create such a file.

Reply:
👍🏻 Removed it.